src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertFromJsonCommand.cs

```diff
@@ -36,6 +36,13 @@ public class ConvertFromJsonCommand : Cmdlet
         [Parameter()]
         public SwitchParameter AsHashtable { get; set; }
 
+        /// <summary>
+        /// Gets or sets the maximum depth the json input is allowed to have. By default, it is 1024. For no maximum, set to 0.
+        /// </summary>
+        [Parameter]
```

Comment:
```suggestion
        [Parameter()]
```

Reply:
Is this common in the cmdlet files? I was under the impression that this is only common in PowerShell because the syntax is mandated there, but in C# the parentheses aren't usually added unless needed.

Reply:
Required? No. Convention? Yes. At least with in this and other Web Cmdlets

Reply:
Just checking; haven't seen it in a lot of other commands in the repo, is all. Thanks! 🌸